frontend/model/contracts/identity.js

```diff
@@ -169,18 +181,18 @@ sbp('chelonia/defineContract', {
       validate: (data, { state, getters }) => {
         objectOf({
           contractID: string,
-          hidden: boolean
+          visible: boolean
         })(data)
         if (!getters.ourDirectMessages[data.contractID]) {
           throw new TypeError(L('Not existing direct message.'))
         }
       },
       process ({ data }, { state, getters }) {
-        Vue.set(state.chatRooms[data.contractID], 'hidden', data.hidden)
+        Vue.set(state.chatRooms[data.contractID], 'visible', data.visible)
       },
       sideEffect ({ data }) {
-        const { contractID, hidden } = data
-        hidden ? leaveChatRoom({ contractID }) : sbp('chelonia/contract/sync', contractID)
+        const { contractID, visible } = data
+        visible ? sbp('chelonia/contract/sync', contractID) : leaveChatRoom({ contractID })
```

Comment:
> Why are we leaving the chatroom if its visibility is set to hidden?

As far as I checked the DM in Slack, I remember we don't get any sounds or notifications. And I wanted to free users from syncing the chatroom contract if they don't think they need the chatroom anymore.

I was going to implement the following feature which is in Slack.
![image](https://github.com/okTurtles/group-income/assets/57976479/7f9b8b74-a87c-424f-b54b-6db0578b8214)
Since we don't have design for this feature yet, I will remove all stuff related to this attribute visibility. Is that fine?

Reply:
You don’t need to remove everything, just all of the logic related to it. The visibility attribute should not affect any behaviors like leaving or joining chatrooms. So logic related to that should be removed. The only thing in the contracts related to the visibility boolean attribute is turning it on and turning it off. That’s it. Nothing else. If we need it in the future, the UI will handle the rest. It must not affect contract logic.

Reply:
And also the notifications and the sounds should work only when the `visibile` attribute is `true`. Right?

Reply:
No. They should always work. Remove all logic related to visible. Only leave logic to turn it on and off and remove everything else. We have not designed anything related to it, so please don’t make any assumptions with the logic.

Reply:
Got it.